<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogIn</class>
 <widget class="QMainWindow" name="LogIn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri Light</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(34, 40, 49)</string>
   </property>
   <widget class="QWidget" name="login_background" native="true">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>120</y>
      <width>391</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 238);
border-radius: 25px;
</string>
    </property>
    <widget class="QLabel" name="login_text">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri Light</family>
       <pointsize>24</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Log In</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="username_input">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(0, 173, 181);
border-radius: 12px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="username_text">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri Light</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password_input">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>260</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(0, 173, 181);
border-radius: 12px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="password_text">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>230</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri Light</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login_button">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>320</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 173, 181);
border-radius: 12px;
color: rgb(238, 238, 238)
</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="member_verif_text">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>370</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri Light</family>
      </font>
     </property>
     <property name="text">
      <string>Not a member?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="signup_button">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>370</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri Light</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
color: rgb(0, 173, 181);</string>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
